src/js/components/Box/Box.js

```diff
@@ -64,6 +64,33 @@ const Box = forwardRef(
 
     let background = backgroundProp;
 
+    useEffect(() => {
+      if (
+        process.env.NODE_ENV !== 'production' &&
+        backgroundProp &&
+        theme.global.deprecated?.backgrounds
+      ) {
+        let deprecatedBackground;
+        if (typeof backgroundProp === 'string') {
+          deprecatedBackground = theme.global.deprecated.backgrounds.find(
+            (item) => item.name === backgroundProp,
+          );
+        } else if (typeof backgroundProp === 'object') {
+          deprecatedBackground = theme.global.deprecated.backgrounds.find(
+            (item) =>
+              item.name === backgroundProp.image ||
+              item.name === backgroundProp.color,
+          );
+        }
+        if (deprecatedBackground) {
+          console.warn(
+            deprecatedBackground.message ||
+              `The background ${deprecatedBackground.name} is deprecated.`,
```

Comment:
Should this also say something like
```suggestion
              `The background ${deprecatedBackground.name} is deprecated. It will be removed in the next major theme version`,
```
It's a little awkward that this code doesn't really know if/when it will go away since the deprecated color is defined in some theme and not grommet.

Reply:
Yeah I was thinking for the hpe theme we would use our own messaging and the fallback message would just be super basic